src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -45,17 +44,13 @@ import {
 import { FormikHelpers } from "formik"
 import {
   ShippingContext,
-  useComputeOrderContext,
-} from "Apps/Order/Routes/Shipping2/ShippingContext"
-import { createUserAddress } from "Apps/Order/Mutations/CreateUserAddress"
-import { useSystemContext } from "System/useSystemContext"
-import { setShipping } from "Apps/Order/Mutations/SetShipping"
-import {
-  FulfillmentType,
-  ShippingAddressFormValues,
-} from "Apps/Order/Routes/Shipping2/shippingUtils"
+  useComputeShippingContext,
+} from "Apps/Order/Routes/Shipping2/support/ShippingContext"
+
+import { FulfillmentType } from "Apps/Order/Routes/Shipping2/support/shippingUtils"
 import { usePrevious } from "Utils/Hooks/usePrevious"
 import { useOrderTracking } from "Apps/Order/Utils/useOrderTracking"
+import { log, warn } from "console"
```

Comment:
console.log, console.warn; we don't pluck these anywhere else, so we should start now

Reply:
ah yes, that was from earlier debugging and i already removed it - got advice from SO that sometimes when console output doesn't appear in jest you can hack around it (not sure how) by importing from console (but maybe logger would work just as well).